### ISSUE TITLE
Problematic JPEG image

### ISSUE THREAD

**Issue by marcuslinke** [haraldk/TwelveMonkeys#152]
Hi Harald,

after running Twelvemonkey in our production env over a year now i still found a single problematic JPEG that can't be read:

```
java.awt.color.CMMException: General CMM error517
	at sun.awt.color.CMM.checkStatus(CMM.java:131)
	at sun.awt.color.ICC_Transform.<init>(ICC_Transform.java:88)
	at java.awt.image.ColorConvertOp.filter(ColorConvertOp.java:516)
	at com.twelvemonkeys.imageio.plugins.jpeg.JPEGImageReader.readImageAsRasterAndReplaceColorProfile(Unknown Source)
	at com.twelvemonkeys.imageio.plugins.jpeg.JPEGImageReader.read(Unknown Source)
```

https://cloud.githubusercontent.com/assets/2578608/8723066/1572935c-2bcb-11e5-90fb-414496a63f7d.jpg

It would be nice if you could take a look at it. 

Cheers

**Comment by haraldk** [comment]
Hi Marcus,

Already fixed in the latest master, I think also included in 3.1-bugfix and 3.1.1 release. 

Which version of Java and TwelveMonkeys ImageIO are you using?

-- 
Harald K

**Comment by marcuslinke** [comment]
Hi Harald,

thanks for the prompt reply! At the moment I'm am tied to Java 6 and TM 3.0.2 but this will change in the near future. If it will work with TM 3.x its OK. I just wanted to give you feedback. 

Keep up the good work and thanks for this great library!

Marcus
